README.md

```diff
@@ -140,16 +173,44 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+convention
+REST is a for how information should be communicated between a browser and a server.
+Seven RESTful routes: INDEX, SHOW, NEW, CREATE, EDIT, UPDATE, DELETE.
```

Comment:
:+1:  Also, in Rails, a RESTful route provides a mapping between HTTP verbs, controller actions, and (implicitly) CRUD operations in a database.